celery/backends/base.py

```diff
@@ -647,11 +648,32 @@ def _forget(self, task_id):
 
     def _store_result(self, task_id, result, state,
                       traceback=None, request=None, **kwargs):
```

Comment:
is the changes in this file tested? report shows coverage decrease

Reply:
@auvipy I'm working through them now.  There's some tests failing that I thought were unrelated to this change, but i'll double check.

Reply:
@auvipy Ok, it looks pretty clean now. The only tests failing are for LoggingProxy issues which looks unrelated.

As an aside, this is my first change to Celery, and it's a big project to dig into. If you or anyone sees a better or more graceful way to accomplish this, please suggest.